source/photo.html

```diff
@@ -79,9 +79,9 @@ <h2 class="photo-card__name">Неродные просторы</h2>
           <div class="photo-card">
             <a class="photo-card__link">
               <picture class="photo-card__picture">
-                <source media="(min-width: 1200px)" srcset="img/photo-cactus-desktop@1x.jpg" srcset="img/photo-cactus-desktop@2x.jpg">
-                <source media="(min-width: 768px)" srcset="img/photo-cactus-tablet@1x.jpg" srset="img/photo-cactus-tablet@2x.jpg">
-                <img class="photo-card__image" src="img/photo-cactus-mobile@1x.jpg" srcset="img/photo-cactus-mobile@2x.jpg" width="400" height="280" alt="Фото растительности Седоны">
+                <source media="(min-width: 1200px)" srcset="img/photo-cactus-desktop@1x.jpg 1x, img/photo-cactus-desktop@2x.jpg 2x">
+                <source media="(min-width: 768px)" srcset="img/photo-cactus-tablet@1x.jpg 1x, img/photo-cactus-tablet@2x.jpg 2x">
+                <img class="photo-card__image" src="img/photo-cactus-mobile@1x.jpg, img/photo-cactus-mobile@2x.jpg 2x" width="400" height="280" alt="Фото растительности Седоны">
```

Comment:
Илья, вот здесь в `img` ошибка, хотя я приводил правильный пример. В `src` указывается путь к файлу со стандартной плотностью пикселей - для 1x, а в в `srcset` путь к файлу с двойной плотностью - для 2x. Должно быть вот так:
```suggestion
                <img class="photo-card__image" src="img/photo-sunset-mobile@1x.jpg" srcset="img/photo-sunset-mobile@2x.jpg 2x" width="280" height="280" alt="Фото растительности Седоны">
```

И обрати внимание на значение `width` - оно должно соответствовать размерам изображения мобильной версии. Но можно вообще width и height для кадрируемых изображений не указывать.

А сейчас наблюдаем вот такую картину: http://prntscr.com/u1p3lc

У остальных картинок исправь тоже.